commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/AbstractMultipleGenerator.java

```diff
@@ -0,0 +1,30 @@
+package io.sphere.sdk.annotations.processors.generators;
+
+import com.squareup.javapoet.JavaFile;
+import com.squareup.javapoet.TypeSpec;
+
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.Elements;
+import java.util.ArrayList;
+import java.util.List;
+
+abstract class AbstractMultipleGenerator extends BaseAbstractGenerator {
```

Comment:
What about changing the name of this class to sth like `AbstractMultipleFilegenerator`? And adding a short javadoc with an explanation would be nice too 😉